out-adapter-persistence-jpa/src/main/java/gohigher/application/entity/ApplicationRepository.java

```diff
@@ -20,5 +21,13 @@ public interface ApplicationRepository extends JpaRepository<ApplicationJpaEntit
 		+ "WHERE a.userId = :userId "
 		+ "AND FUNCTION('YEAR', p.schedule) = :year "
 		+ "AND FUNCTION('MONTH', p.schedule) = :month")
-	List<ApplicationJpaEntity> findByUserIdAndDate(Long userId, int year, int month);
+	List<ApplicationJpaEntity> findByUserIdAndMonth(Long userId, int year, int month);
+
+	@Query("SELECT a FROM ApplicationJpaEntity a "
+		+ "JOIN FETCH a.processes p "
+		+ "WHERE a.userId = :userId "
+		+ "AND p.schedule >= :startOfDate "
+		+ "AND p.schedule < :endOfDate"
+	)
+	List<ApplicationJpaEntity> findByUserIdAndDate(Long userId, LocalDateTime startOfDate, LocalDateTime endOfDate);
```

Comment:
위의 코드들과 통일성을 맞춰보는건 어떨까요?

```suggestion
		+ "AND p.schedule >= :startOfDate "
		+ "AND p.schedule < :endOfDate")
	List<ApplicationJpaEntity> findByUserIdAndDate(Long userId, LocalDateTime startOfDate, LocalDateTime endOfDate);
```